RELEASE.md

```diff
@@ -1,25 +1,19 @@
-# *Eligible Collateral Representation - New Attributes*
+# *CDM support for increase on existing Trade Lot*
 
 _Background_
 
-Through the Collateral Working Group, members have requested two enhancements to the modelling of collateral eligibility to enhance the CDM's capability to support
-additional use cases:
-1. To support the scenario where a legacy collateral schedule has shared criteria for IM and VM, with selected terms applicable to only one single margin type
-2. To prioritise between Collateral Criteria where agency ratings are specified for both assets and issuers
+CDM does not allow for an existing tradeLot to be increased by a delta amount, however it does allow decrease via a delta amount. 
```

Comment:
This release adds support for price quantity changes on an existing `TradeLot`, as described in issue [#2923](https://github.com/finos/common-domain-model/issues/2923).